charts/internal/gvisor/templates/daemonset.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: containerd-gvisor
+  namespace: kube-system
+  labels:
+    app.kubernetes.io/name: containerd-gvisor
+    helm.sh/chart: containerd-gvisor
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: containerd-gvisor
+  template:
+    metadata:
```

Comment:
Can you here add an annotation checksum for the configmap that you mount? Ref https://github.com/gardener/gardener-extension-provider-aws/blob/f9b9f455c6af7b6e915f3f2de4a438cb4f663bdf/charts/gardener-extension-validator-aws/charts/runtime/templates/deployment.yaml#L17, for example.